### PR TITLE
W-15818492 order mgmt guide

### PR DESCRIPTION
**Pull request by tasmoore**

### CODE REVIEW

File: salesforce-oms/1.1/modules/ROOT/pages/order-mgmt-guide.adoc [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+Billing Document: invoice created on the basis of an outbound delivery, to request payment from the customer who ordered the delivery.
+
+== Create a Sales Order
+
+Follow these steps to create a sales order in SAP:
+
+. Search for the *Create Sales Order - VA01* app.
+. Enter *OR* in the *Order Type* field.
+. Press Enter.
+. Enter a value in the *Sold-to Party* field.
```
Comment (tasmoore) [inline salesforce-oms/1.1/modules/ROOT/pages/order-mgmt-guide.adoc]: @nisha-nizamuddin What kind of value should be entered in the field?

File: salesforce-oms/1.1/modules/ROOT/pages/order-mgmt-guide.adoc [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+. Enter a value in the *Ship-to Party* field.
```
Comment (tasmoore) [inline salesforce-oms/1.1/modules/ROOT/pages/order-mgmt-guide.adoc]: @nisha-nizamuddin What kind of value should be entered in the field?

File: salesforce-oms/1.1/modules/ROOT/pages/order-mgmt-guide.adoc [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+. Click the *Display header details* icon and verify that the *Shp. Cond* on the *Shipping* tab is set to *TM - Int. Planning*, which enables SAP to create a freight unit automatically. The freight unit is used when creating the freight order.
+. Enter a value in the Material column of the *All Items* section.
```
Comment (tasmoore) [inline salesforce-oms/1.1/modules/ROOT/pages/order-mgmt-guide.adoc]: @nisha-nizamuddin What kind of value should be entered in the column?

File: salesforce-oms/1.1/modules/ROOT/pages/order-mgmt-guide.adoc [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+. Enter a value in the *Order Quantity* field.
```
Comment (tasmoore) [inline salesforce-oms/1.1/modules/ROOT/pages/order-mgmt-guide.adoc:46]: @nisha-nizamuddin What kind of value should be entered in the field?

File: salesforce-oms/1.1/modules/ROOT/pages/order-mgmt-guide.adoc [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+== Create a Freight Order
+
+Follow these steps to create a freight order in SAP:
+
+. Search for the *Transportation Cockpit* app.
+. Select *ERP Document* and paste the sales order ID into the *Freight Unit Stages* field.
+. Enter *** in the *Vehicle Types* field.
+. Click *Continue*.
+. Select the freight unit of the sales order in the *Freight Unit* column.
+. Select a value in the *Trucks* field.
```
Comment (tasmoore) [inline salesforce-oms/1.1/modules/ROOT/pages/order-mgmt-guide.adoc]: @nisha-nizamuddin What kind of value should be entered in the field?

File: salesforce-oms/1.1/modules/ROOT/pages/order-mgmt-guide.adoc [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+. Click *Save*.
+
+== Create an Outbound Delivery
+
+Follow these steps to create an outbound delivery in SAP:
+
+. Click the *Create Deliveries* tab, and then click the *Document Flow* tab.
+. Check the box for each document that is related to the freight order.
+. Open SAP S/4HANA in a new tab and search for the *Change Outbound Delivery* app.
+. Enter a value in the *Outbound Delivery* field, and then press Enter.
```
Comment (tasmoore) [inline salesforce-oms/1.1/modules/ROOT/pages/order-mgmt-guide.adoc]: @nisha-nizamuddin What kind of value should be entered in the field?

File: salesforce-oms/1.1/modules/ROOT/pages/order-mgmt-guide.adoc [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+. Click *Save*.
+
+== View Your Documents
+
+Follow these steps in SAP to view the documents you created:
+
+. Search for the *Manage Sales Order Version 2* app, and then filter by the sales order ID.
+. Click the sales order ID in the *Sales Orders* table to open the order details.
+. Click *Process Flow* to view the documents you created.
+
```
Comment (tasmoore) [inline salesforce-oms/1.1/modules/ROOT/pages/order-mgmt-guide.adoc:100]: @nisha-nizamuddin Do we want to include a **Next Steps** section to let the user know where to go once they've completed the steps in this guide?

File: salesforce-oms/1.1/modules/ROOT/pages/index.adoc [hunk 8]
```diff
@@ -11,7 +11,7 @@ The https://anypoint.mulesoft.com/exchange/0b4cad67-8f23-4ffe-a87f-ffd10a1f6873/
 
 === Salesforce OMS to ERP Integration
 
-The https://anypoint.mulesoft.com/exchange/0b4cad67-8f23-4ffe-a87f-ffd10a1f6873/mulesoft-accelerator-for-salesforce-oms/minor/{salesforce-oms-version}/pages/Use%20case%202%20-%20Salesforce%20OMS%20to%20ERP%20Integration/[Salesforce OMS to ERP Integration] use case lets you manage orders received via omnichannel sources within Salesforce Order Management System (OMS).
+The https://anypoint.mulesoft.com/exchange/0b4cad67-8f23-4ffe-a87f-ffd10a1f6873/mulesoft-accelerator-for-salesforce-oms/minor/{salesforce-oms-version}/pages/Salesforce%20OMS%20to%20ERP%20Integration/[Salesforce OMS to ERP Integration] use case lets you manage orders received via omnichannel sources within Salesforce Order Management System (OMS).
```
Comment (Cristian-Venticinque) [inline salesforce-oms/1.1/modules/ROOT/pages/index.adoc]:
```suggestion
Manage orders received via omnichannel sources within Salesforce Order Management System (OMS) with the https://anypoint.mulesoft.com/exchange/0b4cad67-8f23-4ffe-a87f-ffd10a1f6873/mulesoft-accelerator-for-salesforce-oms/minor/{salesforce-oms-version}/pages/Salesforce%20OMS%20to%20ERP%20Integration/[Salesforce OMS to ERP Integration] use case.
```
Focus on user/task

File: salesforce-oms/1.1/modules/ROOT/pages/order-mgmt-guide.adoc [hunk 9]
```diff
@@ -0,0 +1,103 @@
...
+Copy the sales order ID, which you'll need in the next steps.
+
+== Create a Freight Order
+
+Follow these steps to create a freight order in SAP:
+
+. Search for the *Transportation Cockpit* app.
+. Select *ERP Document* and paste the sales order ID into the *Freight Unit Stages* field.
+. Enter *** in the *Vehicle Types* field.
```
Comment (Cristian-Venticinque) [inline salesforce-oms/1.1/modules/ROOT/pages/order-mgmt-guide.adoc]:
```suggestion
. Enter `***` in the *Vehicle Types* field.
```
To make it clear that *** is the necessary input in this field, (unless I'm not understanding this properly).

Or, if * is the input, use `*`. If the user needs to type this it shouldn't be bold. Also, confirm in a local build that the text displays as intended.

File: salesforce-oms/1.1/modules/ROOT/pages/order-mgmt-guide.adoc [hunk 10]
```diff
@@ -0,0 +1,103 @@
+= SAP S/4HANA Order Management Guide
+:salesforce-oms-version: {page-component-version}
+
+== Overview
+
+Before you implement the https://anypoint.mulesoft.com/exchange/0b4cad67-8f23-4ffe-a87f-ffd10a1f6873/mulesoft-accelerator-for-salesforce-oms/minor/{salesforce-oms-version}/pages/B2B%20EDI%20Order%20Lifecycle%20Automation/[Salesforce OMS to ERP Integration] use case, you must create a sales order, freight order, outbound delivery, and invoice in SAP.
```
Comment (Cristian-Venticinque) [inline salesforce-oms/1.1/modules/ROOT/pages/order-mgmt-guide.adoc]:
```suggestion 
This guide helps you create a sales order, freight order, outbound delivery, and invoice in SAP so that you can implement the https://anypoint.mulesoft.com/exchange/0b4cad67-8f23-4ffe-a87f-ffd10a1f6873/mulesoft-accelerator-for-salesforce-oms/minor/{salesforce-oms-version}/pages/B2B%20EDI%20Order%20Lifecycle%20Automation/[Salesforce OMS to ERP Integration] use case.
```
After reading the whole article, I'd suggest rewording like this.

**Review by Cristian-Venticinque — APPROVED** [review]
Please check comments before merging.

File: salesforce-oms/1.1/modules/ROOT/pages/order-mgmt-guide.adoc (hunk 9 above)
Comment (tasmoore) [reply salesforce-oms/1.1/modules/ROOT/pages/order-mgmt-guide.adoc]: It should be a single *, so I'll remove the bold.

File: salesforce-oms/1.1/modules/ROOT/pages/order-mgmt-guide.adoc [hunk 11]
```diff
@@ -0,0 +1,103 @@
...
+A contractual agreement between a sales organization and a sold-to party to deliver products or provide a service at a set price, quantity, and time.
+* *Freight Order*
++
+The order that a carrier or shipper plans to execute, containing the logistical processing plan, vehicle departure times, and order execution data.
+* *Outbound Delivery Order*
++
+Contains the data that is required to trigger and monitor the outbound delivery process.
+* *Billing Document*
++
+The invoice that is created for an outbound delivery and that requests payment from the customer who placed the order.
```
Comment (failup) [inline salesforce-oms/1.1/modules/ROOT/pages/order-mgmt-guide.adoc]: Suggestion: The invoice that is created for an outbound delivery and requests payment from the customer who placed the order.

File: salesforce-oms/1.1/modules/ROOT/pages/order-mgmt-guide.adoc [hunk 12]
```diff
@@ -0,0 +1,103 @@
...
+
+* Manual Transportation Planning (6W2)
+* Transportation Execution (6W3)
+* Freight Settlement (6W4)
+
+For more information about how to complete the ticket, see the https://userapps.support.sap.com/sap/support/knowledge/en/3122430[Scope item activation request for CBC provisioned S/4HANA Cloud Essentials tenant] knowledge base article on the SAP support site.
+
+=== Generated SAP Documents
+
+Completing the steps in this guide generates following SAP documents:
```
Comment (failup) [inline salesforce-oms/1.1/modules/ROOT/pages/order-mgmt-guide.adoc]: Suggestion: the following SAP documents

File: salesforce-oms/1.1/modules/ROOT/pages/order-mgmt-guide.adoc [hunk 13]
```diff
@@ -0,0 +1,103 @@
...
+. Click *Save*.
+
+== Create an Outbound Delivery
+
+Follow these steps to create an outbound delivery in SAP:
+
+. Click the *Create Deliveries* tab and then click the *Document Flow* tab.
+. Select the box for each document that is related to the freight order.
+. Open SAP S/4HANA in a new tab and search for the *Change Outbound Delivery* app.
+. Enter outbound delivery number from the *Create Deliveries* tab in the *Outbound Delivery* field and then press Enter.
```
Comment (failup) [inline salesforce-oms/1.1/modules/ROOT/pages/order-mgmt-guide.adoc]: suggestion: Enter the outbound...

File: salesforce-oms/1.1/modules/ROOT/pages/order-mgmt-guide.adoc [hunk 14]
```diff
@@ -0,0 +1,103 @@
...
+
+Copy the sales order ID, which you'll need in the next steps.
+
+== Create a Freight Order
+
+Follow these steps to create a freight order in SAP:
+
+. Search for the *Transportation Cockpit* app.
+. Select *ERP Document* and paste the sales order ID into the *Freight Unit Stages* field.
+. Enter * in the *Vehicle Types* field.
```
Comment (Cristian-Venticinque) [inline salesforce-oms/1.1/modules/ROOT/pages/order-mgmt-guide.adoc]:
```suggestion
. Enter `*` in the *Vehicle Types* field.
```

**Review by Cristian-Venticinque — APPROVED** [review]
